test/test_v2/test_common.c

```diff
@@ -204,16 +258,92 @@ void free_test_params(test_params *params) {
     }                                 
 } 
 
-void pmixt_pre_init(int argc, char **argv, test_params *params) {
+void free_validation_params(validation_params *val_params) { 
+    //placeholder for if/when we have something that needs to be freed
+    ;
+} 
+
+void set_client_argv(test_params *params, char ***argv)
```

Comment:
Lets pass this info through parameters.
No need to separate params and validation params

Reply:
will be fixed in a subsequent commit.